docs/SONATA_DEVELOPER_GUIDE.md

```diff
@@ -830,11 +830,11 @@ Table 2: Layout of the file format for describing edges.
 
 ##### Edges - Required Attributes
 
-**edge_type_id** -  Like the node_type_id, this is a unique integer to associate an edge to an edge type.  An edge type has associated attributes, and an edge inherits attributes from its edge type.  Attributes associated to an edge override attributes inherited from the edge type.  edge_type_ids need not be ordered or contiguous, but must be unique.  A reference implementation might be to use a id-value store, such as a dictionary to associate a edge_type_id with its associated attribute values.
+**edge_type_id** [INT32]-  Like the node_type_id, this is a unique integer to associate an edge to an edge type.  An edge type has associated attributes, and an edge inherits attributes from its edge type. Attributes associated to an edge override attributes inherited from the edge type. edge_type_ids need not be ordered or contiguous, but must be unique. If no base node type is used for a particular edge, `-1` value is used. A reference implementation might be to use a id-value store, such as a dictionary to associate a edge_type_id with its associated attribute values.
```

Comment:
Just want to reiterate about what I said about node_type_id. I think a node_type_id=-1 may be a reasonable request in the future, but at this stage I would rather not make changes to requirements until we've had time discuss them (also we have to consider the deadlines).

Reply:
But what should we put here as a value when we don't have node_type.csv (edge_type.csv) files ?
I don't see why these 2 files should be mandatory. We won't have that for our circuits and we put these files in the spec as a convenience for your use cases.
What is your suggestion for our case ?